docroot/modules/custom/va_gov_backend/src/EventSubscriber/TableAuditViewsEventSubscriber.php

```diff
@@ -5,7 +5,7 @@
 use Drupal\Core\Render\RendererInterface;
 use Drupal\Core\Link;
 use Drupal\Core\Url;
-use Drupal\entity_reference_revisions\EntityReferenceRevisionsOrphanPurger;
+use Drupal\node\Entity\Node;
```

Comment:
Do we want NodeInterface here?
```suggestion
use Drupal\node\NodeInterface;
```

Reply:
Yes. Fixed.